CMS-Enterprise/sbom-harbor
# feat: set up Clap CLI structure

## Summary

This commit sets up the CLI scaffolding.

### Added

Command handling structure.

## How to test

- Pull the repo
- Either run `cargo build` and then run the executable from `target/debug/harbor-cli` OR
- Run the main function from your ide
- Since no subcommand was specified, you should see `command not found` printed to stdout.
